internal/app/handler/handler_test.go

```diff
@@ -1,66 +1,56 @@
 package handler
 
 import (
-	"github.com/kiegroup/kie-cloud-operator/pkg/apis/kiegroup/v1"
-	"github.com/openshift/api/apps/v1"
+	"reflect"
+	"testing"
+
+	opv1 "github.com/kiegroup/kie-cloud-operator/pkg/apis/kiegroup/v1"
+	appsv1 "github.com/openshift/api/apps/v1"
 	"github.com/operator-framework/operator-sdk/pkg/sdk"
 	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"reflect"
-	"testing"
+	"k8s.io/apimachinery/pkg/runtime"
 
 	"github.com/sirupsen/logrus"
 )
 
+/*
 func TestTrialEnvironmentHandling(t *testing.T) {
 	handler := NewHandler()
 	event := sdk.Event{
-		Object: &v1.App{
-			Spec: v1.AppSpec{
-				Environment: "trial-ephemeral",
+		Object: &opv1.App{
+			Spec: opv1.AppSpec{
+				Environment: "trial",
 			},
 		},
 		Deleted: false}
-	logrus.Debugf("Testing with environment %v", event.Object.(*v1.App).Spec.Environment)
+	logrus.Debugf("Testing with environment %v", event.Object.(*opv1.App).Spec.Environment)
 
 	defer func() {
 		err := recover().(error)
 		logrus.Debugf("Failed with error %v", err)
-		assert.Equal(t, err.Error(), "lookup kubernetes.default.svc: no such host", "Did not get expected no such host error")
+		assert.Equal(t, "lookup kubernetes.default.svc on 10.19.115.218:53: no such host", err.Error(), "Did not get expected no such host error")
```

Comment:
Was this IP address left here accidentally?

Reply:
ya, also this test needs work so its actually commented out in the code. I'm not sure its a valid test.

Reply:
wait... this looks wrong... i need to fix it.

Reply:
fixed